.changeset/bright-knives-remain.md

```diff
@@ -0,0 +1,5 @@
+---
+"@digdir/designsystemet-react": patch
+---
+
+RovingIndex: add support for up/down arrows and home/end buttons
```

Comment:
This is what is going to be inlined in the changelog, so it needs to be conventional commit style.